ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -315,9 +316,10 @@ class WalletHomeScreen extends React.Component<Props, never> {
       <Button
         block={true}
         onPress={
-          pot.isSome(potWallets)
-            ? this.props.navigateToPaymentScanQrCode
-            : undefined
+          this.props.navigateToCIEOK
```

Comment:
you can revert all these changes.
For test purpose I will build an ad-hoc version that allows to display this screen

Reply:
Yes,  it was just the fastest way I had already available

Reply:
Yes I know but this PR will be merged and when it is merged that version is available to all beta tester. If we miss something we could publish a wrong version of the app